components/components.html

```diff
@@ -8,296 +8,312 @@
+                    src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522divider%2522%253E%253C%252Fdiv%253E"
+                    style="border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </section>
+            <hr>
+            <section id="pagination">
+                <h1>Pagination</h1>
+                <p>A Pagination components helps you to navigate on different pages.</p>
+                <ul class="pagination">
+                    <li class="page-item disabled"><a class="page-link" href="#1">Previous</a></li>
+                    <li class="page-item"><a class="page-link" href="#2">1</a></li>
+                    <li class="page-item"><a class="page-link" href="#3">2</a></li>
+                    <li class="page-item"><a class="page-link" href="#4">3</a></li>
+                    <li class="page-item"><a class="page-link" href="#5">Next</a></li>
+                </ul>
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cul%2520class%253D%2522pagination%2522%253E%250A%2520%2520%253Cli%2520class%253D%2522page-item%2520disabled%2522%253E%253Ca%2520class%253D%2522page-link%2522%2520href%253D%2522%25231%2522%253EPrevious%253C%252Fa%253E%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522page-item%2522%253E%253Ca%2520class%253D%2522page-link%2522%2520href%253D%2522%25232%2522%253E1%253C%252Fa%253E%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522page-item%2522%253E%253Ca%2520class%253D%2522page-link%2522%2520href%253D%2522%25233%2522%253E2%253C%252Fa%253E%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522page-item%2522%253E%253Ca%2520class%253D%2522page-link%2522%2520href%253D%2522%25234%2522%253E3%253C%252Fa%253E%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522page-item%2522%253E%253Ca%2520class%253D%2522page-link%2522%2520href%253D%2522%25235%2522%253ENext%253C%252Fa%253E%253C%252Fli%253E%250A%253C%252Ful%253E"
+                    style="border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </section>
+            <hr>
+            <section id="breadcrumb">
+                <h1>Breadcrumb</h1>
+                <p>Breadcrumbs are a secondary navigation aid that helps users easily understand the relation between
+                    their location on a page (like a product page) and higher level pages (a category page, for
+                    instance)</p>
+                <ul class="breadcrumb">
```

Comment:
Implement the same at top of components page and keep it here as well. Goal is to keep it here as part of component but also use it in our website to beautify it

Reply:
hey @imabhinavagarwal , Do you want website to look like
![image](https://user-images.githubusercontent.com/96467990/194031766-176bf7fd-6dfe-4aaf-8e2a-84397e479ce4.png)
# Also tell if you want breadcrumb to be fixed  on top of website or not. 
# Do you want any javascript feature to be added to breadcrumb or not . Example should it get updated when you click to any component.
 :)